editor/editor_interface.cpp

```diff
@@ -555,6 +588,13 @@ void EditorInterface::_quick_open(const String &p_file_path, const Callable &p_c
 	_call_dialog_callback(p_callback, p_file_path, "quick open");
 }
 
+void EditorInterface::_create_dialog_item_selected(const bool p_is_cancelled, const Callable &p_callback) {
+	create_dialog->disconnect(SNAME("create"), callable_mp(this, &EditorInterface::_create_dialog_item_selected));
+	create_dialog->disconnect(SNAME("canceled"), callable_mp(this, &EditorInterface::_create_dialog_item_selected));
+	StringName selected_type = create_dialog->get_selected_type();
```

Comment:
```suggestion
	const String selected_type = create_dialog->get_selected_type();
```
The method returns a String.

Also it would be nice to avoid calling `get_selected_type()` with `p_is_cancelled`, as it's not going to be used anyway.